apps/condo/domains/organization/integrations/sbbol/sync/syncClientAccounts.js

```diff
@@ -0,0 +1,83 @@
+const dayjs = require('dayjs')
+
+const conf = require('@open-condo/config')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { getLogger } = require('@open-condo/keystone/logging')
+const { get, isEmpty } = require('lodash')
+
+const { initSbbolFintechApi } = require('../SbbolFintechApi')
+const { dvSenderFields } = require('../constants')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const { RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { ISO_CODES_FOR_SBBOL } = require('@condo/domains/common/constants/currencies')
+
+const logger = getLogger('sbbol/syncClientAccounts')
+const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
+
+const syncWithBankAccount = async (context, client, organizationId) => {
+    for (let accountIndex = 0; accountIndex <= client.accounts.length - 1; accountIndex++) {
+        const currentAccount = client.accounts[accountIndex]
+        const bankAccountDetails = {
+            tin: client.inn,
+            country: RUSSIA_COUNTRY,
+            number: currentAccount.number,
+            currencyCode: (ISO_CODES_FOR_SBBOL[currentAccount.currencyCode]),
+            routingNumber: currentAccount.bic,
+        }
+        const foundAccount = await BankAccount.getAll(context, {
+            ...bankAccountDetails,
+            organization: { id_in: organizationId },
+        }, {
+            sortBy: 'id_ASC',
+            first: 10,
```

Comment:
Why only 10? Is there any business logic related to that constant?

Reply:
It may be worth using `getOne` here because if there is at least one `BankAccount` with the same data, a new one should not be created.